src/OrchardCore.Modules/OrchardCore.Users/Drivers/UserDisplayDriver.cs

```diff
@@ -21,22 +24,32 @@ public class UserDisplayDriver : DisplayDriver<User>
         private readonly UserManager<IUser> _userManager;
         private readonly IRoleService _roleService;
         private readonly IUserRoleStore<IUser> _userRoleStore;
+        private readonly IHttpContextAccessor _httpContextAccessor;
+        private readonly INotifier _notifier;
         private readonly ILogger _logger;
         private readonly IStringLocalizer S;
+        private readonly IHtmlLocalizer H;
 
         public UserDisplayDriver(
             UserManager<IUser> userManager,
             IRoleService roleService,
             IUserRoleStore<IUser> userRoleStore,
+            IHttpContextAccessor httpContextAccessor,
```

Comment:
I don't think we need to inject `IHttpContextAccessor` just checking the `user` will enough

Reply:
Hmm I thought that the User was the current data from the database and model is the one coming from the form. So if we want to check who's the current user we need to get it from the context?